README.md

```diff
@@ -11,6 +11,16 @@ for any bug bounty. We advise you to be careful and experiment on the network at
 
 ## Cmd usage
 
+### installation
+
+```
+git clone https://github.com/bnb-chain/greenfield-cmd.git
+cd greenfield-cmd
+make build
```

Comment:
make PR to develop branch instead of master

Reply:
done